lego/apps/articles/tests/test_articles_api.py

```
@@ -305,3 +317,65 @@ def test_only_one_pinned(self):
         self.assertTrue(self.article.pinned)
         self.assertFalse(pinned.pinned)
         self.assertEqual(Article.objects.filter(pinned=True).count(), 1)
+
+
+def generate_plausible_item(i):
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_plausible_item`

(ANN201)